x-pack/plugins/search_playground/public/components/set_up_connector_panel_for_start_chat.tsx

```diff
@@ -27,64 +28,82 @@ export const SetUpConnectorPanelForStartChat: React.FC = () => {
     refetch: refetchConnectors,
     isLoading: isConnectorListLoading,
   } = useLoadConnectors();
+  const usageTracker = useUsageTracker();
   const handleConnectorCreated = () => {
     refetchConnectors();
     setShowAddedCallout(true);
     setConnectorFlyoutOpen(false);
   };
+  const handleSetupGenAiConnector = () => {
+    usageTracker.click('gen_ai_connector_create');
+    setConnectorFlyoutOpen(true);
+  };
+
+  useEffect(() => {
+    if (connectors?.length) {
+      if (showCallout) {
+        usageTracker.load('gen_ai_connector_added');
+      } else {
+        usageTracker.load('gen_ai_connector_exists');
```

Comment:
I will be interested to know which connectors(LLM models) are being used mostly. Are we tracking it anywhere?
cc: @joemcelroy

Reply:
Good idea, I'll add it

Reply:
just to also update this thread, its likely the kibana connectors themselves have tracking so Yan looking into this too.